policyengine_us/variables/gov/states/de/tax/income/de_pre_exclusions_agi.py

```diff
@@ -3,11 +3,11 @@
 
 class de_pre_exclusions_agi(Variable):
     value_type = float
-    entity = TaxUnit
-    label = "Delaware adjusted gross income before exclusions"
+    entity = Person
+    label = "Delaware individual adjusted gross income before exclusions"
     unit = USD
     definition_period = YEAR
     defined_for = StateCode.DE
 
-    adds = ["de_additions", "adjusted_gross_income"]
+    adds = ["de_additions", "adjusted_gross_income_person"]
```

Comment:
could you do the same thing here as we do in KY and AR for the AGI file 
-> allocate the income of the dependents to the head of the household

Reply:
write a formula and create a unit test with a head + spouse + dependent with income